hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -931,12 +931,28 @@ public FileStatus[] listStatus(final Path f) throws IOException {
           == PrefixMode.BLOB) {
```

Comment:
Resume actions to be made only if incoming path is a parent to one of the configured atomic rename paths to stop parsing of all the List calls.

Reply:
Taken.